Scala/sRAPL/sRAPL.scala

```diff
@@ -0,0 +1,199 @@
+// using scala 3.1
+// using lib jRAPL::jRAPL:1.0
+
+package sRAPL
+
+import jRAPL._
+import java.io._
+
+@main def test(): Unit = {
+given config: Settings = Settings(warmupConfig = WarmupSettings(measureWarmup = true))
+ measureEnergyConsumption("test"){ () =>
+   (0 until 10).foreach{_ =>
+   util.Random.alphanumeric.take(1000000).count(_ == 'A')
+   }
+ } 
+}
+
+def measureEnergyConsumption(
+    label: String
+)(fn: () => Unit)(using config: Settings): Unit = {
+  withFreshSyncMonitor {
+    withConfigPrintStream {
+      val warmupConfig = config.warmupConfig
+      for
+        i <- 0 until warmupConfig.iterations
+        // input = warmupConfig.input.fold(
+        //   throw RuntimeException("Warmup input not defined")
+        // )(_.inputStream())
+      do
+        println(s"$label warmup - iteration $i")
```

Comment:
Is this printing to the configured `PrintStream`?

Reply:
Yes, it should print to the given output, which would be either Standard Output or File. When running benchmarks we print to file, because we redirect stdout to `/dev/null` (writing to console is slow and slows down) and to have there only relevant informations